cogs/streams.py

```diff
@@ -39,9 +39,8 @@ def __init__(self, bot):
     async def twitch(self, stream: str):
         """Checks if twitch stream is online"""
         online = await self.twitch_online(stream)
-        if online is True:
-            await self.bot.say("http://www.twitch.tv/{} "
-                               "is online!".format(stream))
+        if online is not False and online is not "error":
```

Comment:
`if online and online != "error":`
^ that is what this line should read, same with line 313